astroquery/solarsystem/pds/tests/test_pds.py

```diff
@@ -0,0 +1,219 @@
+import pytest
+import os
+import numpy as np
+
+import astropy.units as u
+from ....query import AstroQuery
+
+from astroquery.utils.mocks import MockResponse
+from ... import pds
+
+# files in data/ for different planets
+DATA_FILES = {'Uranus': 'uranus_ephemeris.html',
+              'Pluto': 'pluto_ephemeris.html',
+              'Neptune': 'neptune_ephemeris.html',
+              'Saturn': 'saturn_ephemeris.html',
+              }
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), "data")
+    return os.path.join(data_dir, filename)
+
+
+# monkeypatch replacement request function
+def nonremote_request(self, request_type, url, **kwargs):
+
+    planet_name = kwargs['params']['center_body']
+    with open(data_path(DATA_FILES[planet_name.capitalize()]), "rb") as f:
+        response = MockResponse(content=f.read(), url=url)
+
+    return response
+
+
+# use a pytest fixture to create a dummy 'requests.get' function,
+# that mocks(monkeypatches) the actual 'requests.get' function:
+@pytest.fixture
+def patch_request(request):
+    mp = request.getfixturevalue("monkeypatch")
+
+    mp.setattr(pds.core.RingNodeClass, "_request", nonremote_request)
+    return mp
+
+
+# --------------------------------- actual test functions
+
+def test_parse_result(patch_request):
+    q = pds.RingNode()
+    # need _last_query to be defined
+    q._last_query = AstroQuery('GET', 'http://dummy')
+    with pytest.raises(ValueError):
```

Comment:
It's good to test the exception message because the expected message also conveys the purpose of the test.